tests/e2e/test_rosacli_machine_pool.go

```diff
@@ -660,12 +660,19 @@ var _ = Describe("Create machinepool",
 				Expect(err).ToNot(HaveOccurred())
 
 				By("Create temporary account-roles for instance type list")
-				accountRoles, err := ph.PrepareAccountRoles(rosaClient, "test-55979",
-					false,
-					clusterConfig.Version.RawID,
-					clusterConfig.Version.ChannelGroup,
-					"", "")
+				namePrefix := "test-55979"
+				_, err = rosaClient.OCMResource.CreateAccountRole("--mode", "auto",
+					"--prefix", namePrefix,
+					"--version", clusterConfig.Version.RawID,
```

Comment:
note: in order for this to pass locally I had to manually set it to version 4.17, as the `RawID` was `4.17.0-rc.1` which caused the test to fail. I'm assuming that that's fine in the actual pipelines, but it I figure it's worth bringing up here.

Reply:
Is it possible to truncate the version to just 4.x?

Reply:
I think we want this to test the account roles against the latest possible version, it's just that there was a mismatch locally between what account roles were available and what's available in the repo/pipeline, so it should be fine to leave as is? If not I'd consider removing the `--version` flag entirely from this call and have the command choose the default version instead. Will wait to hear more feedback on this

Reply:
I believe it should be failed using 'clusterConfig.Version.RawID' as the --version value. `rosa create account-roles` only accept the version with format Major.Minor, like 4.17, 4.16 
You may need to do some processing on the  version value to meet the format. Maybe 'common.SplitMajorVersion' can be used or anything other else way you think better.

Reply:
looks like `common.SplitMajorVersion` is how the previous method handled the issue, I will use this as well.